### PR TITLE
Adds void return type to generated snippets

### PR DESCRIPTION
**Pull request by carlos-granados**
Add `void` return type to the snippets that are generated when a step definition is missing. Even though this return type does not offer any runtime advantage and is of limited use for static analysis, it provides completeness if, for example, we are calculating the type coverage of our application

### CODE REVIEW

**Comment by pierresh** [comment]
It is a good idea, PHPStan always reminds me that new methods have no return type specified.

**Review by acoulton — APPROVED** [review]
LGTM

**Comment by acoulton** [comment]
@carlos-granados I see this still needs a second review - are you able to review your own PR?

**Comment by carlos-granados** [comment]
@acoulton no, I am unable to approve a PR that I created myself. @everzet how do we solve this problem? It is going to be an issue if @acoulton and myself start working on pending issues...

**Review by stof — APPROVED** [review]

**Pull request merged**
